react/features/video-quality/constants.js

```diff
@@ -19,6 +19,16 @@ export const VIDEO_QUALITY_LEVELS = {
     NONE: 0
 };
 
+/**
+ * Indicates unlimited video quality.
+ */
+export const VIDEO_QUALITY_UNLIMITED = 99999;
```

Comment:
Yes. Actually it turned out that the bridge was not handling -1 well. But I think this is fixed now and I should update it back to  -1.

The reason that this is not in the enum is because the enum is used in places dynamically and usually we are sorting it and using it to compare the height of the videos with the values. Basically if we have -1 there this whole approach needs to change and will introduce complications. In addition to this, since unlimited is kind of a special constraints, I think it is good idea to be separate.

Reply:
Ok makes sense about not having it as an enum.